Sources/MongoSwift/MongoClient.swift

```diff
@@ -304,6 +304,106 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+     /**
+      * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+      * system collections and the "config", "local", and "admin" databases.
+      * - Parameters:
+      *   - pipeline: An array of aggregation pipeline stages to apply to the events returned by the change stream.
+      *   - options: An optional `ChangeStreamOptions` to use when constructing the change stream.
+      *   - session: An optional `ClientSession` to use with this change stream.
+      * - Returns: a `ChangeStream` on all collections in all databases in a cluster.
+      * - Throws:
+      *   - `ServerError.commandError` if an error occurs on the server while creating the change stream.
+      *   - `ServerError.commandError` if the pipeline passed is invalid.
```

Comment:
I think this error case can be omitted since it's covered under the "error occurs on the server while creating the change stream".